modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,574 @@
+  nodeRef:
+    kind: Machine
+    name: fe80::4adf:37ff:feb0:8aa1%ens1f1.372
+    uid: acca4411-af0d-4387-b73e-52b2484295ad
+  phase: Running
+  providerStatus:
+    apiVersion: machine.openshift.io/v1beta1
+    conditions:
+    - lastProbeTime: "2020-04-20T16:53:50Z"
+      lastTransitionTime: "2020-04-20T16:53:50Z"
+      message: machine successfully created
+      reason: MachineCreationSucceeded
+      status: "True"
+      type: MachineCreation
+    instanceId: i-0fdb85790d76d0c3f
+    instanceState: stopped
+    kind: Machine
+----
+
+. Change the `metadata.name` field to a new name.
++
+It is recommended to keep the same base name as the old machine and change the ending number to the next available number. In this example, `clustername-8qw5l-master-0` is changed to `clustername-8qw5l-master-2`.
++
+For example:
++
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+metadata:
+  ...
+  name: examplecluster-control-plane-3
+  ...
+----
+
+.. Remove the `spec.providerID` field:
++
+[source,yaml]
+----
+  providerID: baremetalhost:///openshift-machine-api/openshift-control-plane-2/3354bdac-61d8-410f-be5b-6a395b056135
+----
+
+.. Remove the `metadata.annotations` and `metadata.generation` fields:
++
+[source,yaml]
+----
+  annotations:
+    machine.openshift.io/instance-state: externally provisioned
+  ...
+  generation: 2
+----
+
+.. Remove the `spec.conditions`, `spec.lastUpdated`, `spec.nodeRef` and `spec.phase' fields:
++
+[source,yaml]
+----
+  lastTransitionTime: "2022-08-03T08:40:36Z"
+message: 'Drain operation currently blocked by: [{Name:EtcdQuorumOperator Owner:clusteroperator/etcd}]'
+reason: HookPresent
+severity: Warning
+status: "False"
+
+type: Drainable
+lastTransitionTime: "2022-08-03T08:39:55Z"
+status: "True"
+type: InstanceExists
+
+lastTransitionTime: "2022-08-03T08:36:37Z"
+status: "True"
+type: Terminable
+lastUpdated: "2022-08-03T08:40:36Z"
+nodeRef:
+kind: Node
+name: openshift-control-plane-2
+uid: 788df282-6507-4ea2-9a43-24f237ccbc3c
+phase: Running
+----
+
+. Ensure that the Bare Metal Operator is available:
++
+[source,terminal]
+----
+$ oc get clusteroperator baremetal
+----
++
+.Example output
+[source,terminal]
+----
+NAME        VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
+baremetal   4.11.3   True        False         False      3d15h
+----
+
+. Delete the machine of the unhealthy member:
++
+[source,terminal]
+----
+$ oc delete machine -n openshift-machine-api clustername-8qw5l-master-2 <1>
+----
++
+If deletion of the machine is delayed for any reason or the command is obstructed and delayed, you can force deletion by removing the machine object finalizer field. 
++
+[IMPORTANT]
+====
+Do not interrupt machine deletion with `control-c`. You must allow the command to proceed to completion. Open a new terminal window to edit and delete the finalizer fields.
+====
++
+[source,terminal]
+----
+$ oc edit machine -n openshift-machine-api clustername-8qw5l-master-0
```

Comment:
naming. And we are deleting control-plane-2, so finalizer should be deleted for him :)